src/_pages/prototyping_boards/electronic-use.md

```diff
@@ -0,0 +1,80 @@
+
+Here are the specifications of this motherboard:
+
+{{ #include ../../_includes/specs.md }}
+
+## Boards categories
+
+Luos boards are organized in 6 categories. Each board belongs to at least one of these categories. Understanding every categories will help to understand how to connect the Luos boards together in order to achieve any robotic system you want.
+
+On each board’s image in the website, one or several small colored symbols are displayed allowing you to easily see the different categories it belongs to:
+
+<img src="/_assets/img/boards_example.png" height="" />
+
+**Below is the list of the six categories:**
+
+|Sensor|Actuator|Communication|
+|:-|:-|:-|
+|![](/_assets/img/sticker-sensor.png)|![](/_assets/img/sticker-actuation.png)|![](/_assets/img/sticker-communication.png)|
+|Sensor boards are able to measure physical world environment.|Actuation boards are able to act on the physical world.|Communication boards (also called gates) are able to share your system’s inputs, outputs and configurations outside of your robot, using a JSON API.|
+
+|Cognition|Interface|Power|
+|:-|:-|:-|
+|![](/_assets/img/sticker-cognition.png)|![](/_assets/img/sticker-interface.png)|![](/_assets/img/sticker-power.png)|
+|Cognition are boards dedicated to execute your code or host your AI.|These boards are built to interact with the user of the machine.|Power boards are able to share their input power source into the Robus wire to feed other boards.|
+
+<a name="plug"></a>
+## Plugging boards together
+
+Luos boards have at least 2 connection ports. All connectors are the same, so that any board can be connected to another one using any of these ports. Just avoid to make a loop circuit, otherwise you will damage the communication between modules.
+
+There is a correct side to plug a cable’s connector to a board. The small tab on the connector must face upward to plug correctly, as shown on the following pictures:
+
+|![Wrong side img](/_assets/img/plug-no.png)|![Right side img](/_assets/img/plug-yes.png)|
+|:-|:-|
+|Wrong side, the upper surface is flat|Right side, the tab is visible on the upper surface|
+
+
+
+## Power management
+
+Luos boards can share their power inputs through the network connection, allowing you to feed other boards. These boards belong to the **power category**.
+All the Luos boards can manage a voltage between 5V and 24V, up to 7A.
+
+In a Luos network, **you can have multiple power category boards**. In this case, **the power board with the highest voltage takes over** and shares its power with other boards.
+
+For example, for a robot using a 12V motor and an USB board: The USB board belongs to the power category, so it can share its 5V into the network's wires. But you need 12V for your motor, so you will have to add a 12V AC plug board in your network to supply the motor. In this case, the USB board doesn’t share its power, only the AC plug board does, because 5V < 12V.
```

Comment:
```suggestion
For example, for a device using a 12V motor and an USB board: The USB board belongs to the power category, so it can share its 5V into the network's wires. But you need 12V for your motor, so you will have to add a 12V AC plug board in your network to supply the motor. In this case, the USB board doesn’t share its power, only the AC plug board does, because 5V < 12V.
```